Source/diablo.cpp

```diff
@@ -157,23 +136,20 @@ int arrowdebug = 0;
 #endif
 /** Specifies whether players are in non-PvP mode. */
 bool gbFriendlyMode = true;
-/** Default quick messages */
-const char *const spszMsgTbl[] = {
+const char *const spszMsgTbl[4] = {

```

Comment:
It would also be nice to merge the key and value in to some sort of struct, maybe even consider something that helps it integrate easy with the keymapper but that will have to wait for a later refactoring.

Reply:
Did both :D